politeiawww/cmd/politeiavoter/params.go

```diff
@@ -38,7 +38,7 @@ var mainNetParams = params{
 
 var testNet3Params = params{
 	Params:              chaincfg.TestNet3Params(),
-	WalletRPCServerPort: "19111",
+	WalletRPCServerPort: "19558",
```

Comment:
Why are you breaking the testnet params?

Reply:
oh, that was a mistake. Will fix! Thanks.